### PR TITLE
Expect gRPC error status in tests

### PR DESCRIPTION
**Pull request by raboof**
Requires https://github.com/akka/akka-grpc/pull/542 first

Fixes https://github.com/akka/akka-grpc/issues/396

~worksWithAGrpcClient seems to timeout for me (both scala and java),
but I haven't figured out yet whether that is a local problem or a
regression.~ (seems to work fine from sbt)

### CODE REVIEW

File: play-testkit/src/test/java/play/grpc/testkit/PlayJavaFunctionalTest.java [hunk 1]
```diff
@@ -49,7 +51,7 @@ public void stopServer() throws Exception {
     }
   }
 
-  private WSResponse wsGet(final String path) throws Exception {
+  private WSResponse wsGqet(final String path) throws Exception {
```
Comment (dwijnand) [inline play-testkit/src/test/java/play/grpc/testkit/PlayJavaFunctionalTest.java]: typo?

**Comment by raboof** [comment]
Fixed typo and formatting, but github won't let me push them:

```
remote: error: GH006: Protected branch update failed for refs/heads/expectGrpcErrorStatus.
remote: error: Required status check "Travis CI - Pull Request" is expected. At least 1 approving review is required by reviewers with write access.
To github.com:playframework/play-grpc
 ! [remote rejected] expectGrpcErrorStatus -> expectGrpcErrorStatus (protected branch hook declined)
```

Any idea how that is to be solved?

File: play-testkit/src/test/java/play/grpc/testkit/PlayJavaFunctionalTest.java (hunk 1 above)
Comment (ignasi35) [reply play-testkit/src/test/java/play/grpc/testkit/PlayJavaFunctionalTest.java]:
```suggestion
  private WSResponse wsGet(final String path) throws Exception {
```

**Comment by dwijnand** [comment]
@raboof fixed: changed the branch protection from "*" (i.e. all branches) to just "master".

File: play-testkit/src/test/java/play/grpc/testkit/PlayJavaFunctionalTest.java [hunk 2]
```diff
@@ -61,15 +63,23 @@ public void returns404OnNonGrpcRequest() throws Exception {
...
   }
 
   @Test
-  public void returns500OnEmptyRequestToAGrpcMethod() throws Exception {
+  public void returnsGrpcInternalErrorOnEmptyRequestToAGrpcMethod() throws Exception {
     final WSResponse rsp = wsGet("/" + GreeterService.name + "/SayHello");
-    assertEquals(500, rsp.getStatus()); // Maybe should be a 426, see #396
+    assertEquals(200, rsp.getStatus());
+    assertEquals(
+      Integer.toString(Status.Code.INTERNAL.value()),
```
Comment (ignasi35) [inline play-testkit/src/test/java/play/grpc/testkit/PlayJavaFunctionalTest.java]: Nice. We should use `Status.Code.INTERNAL.value()` instead of `13` in `Specs2` and `scalatestplus-play` (see https://github.com/playframework/play-grpc/blob/12da1f1b70f3fc2e9ddfb43a585c94d0b37010f6/play-specs2/src/test/scala/play/grpc/specs2/PlaySpecs2Spec.scala#L49)

Happy to do it on a separate PR.
